webapp/components/admin_console/admin_navbar_dropdown.jsx

```diff
@@ -45,6 +49,26 @@ export default class AdminNavbarDropdown extends React.Component {
         TeamStore.removeChangeListener(this.onTeamChange);
     }
 
+    componentDidUpdate() {
+        if (document.body.classList.contains('app__body') && this.state.showAboutModal === false) {
+            // Timeout needed to ensure modal closing animation can fully complete
+            setTimeout(() => {
+                $('body').removeClass('app__body');
+            }, 300);
+        }
+    }
```

Comment:
I used jQuery so that I could add the `app__body` class, so that the modal would render with the correct style. Should I remove the jQuery and try modifying the sass so that it doesn't need that class to render properly?